flang/lib/Parser/message.cpp

```diff
@@ -272,19 +272,51 @@ static llvm::raw_ostream::Colors PrefixColor(Severity severity) {
   return llvm::raw_ostream::SAVEDCOLOR;
 }
 
+// TODO: Make these configurable, based on verbosity level.
+const int MAX_CONTEXTS_EMITTED = 2;
```

Comment:
These should be `static constexpr`, not externally visible `const`,  and initialized with braces.